### ISSUE TITLE
License clarification

### ISSUE THREAD

**Issue by Konstantin8105** [DrTimothyAldenDavis/SuiteSparse#73]
Dear Timothy Davis,
A few time ago, I transpile your package `CSparse` from C programming language to [Golang](https://golang.org/) and store source code into repository [sparse](https://github.com/Konstantin8105/sparse).
If you have license requirements about my repository, then please clarify.

**Comment by Konstantin8105** [comment]
Are you agree to change license for my repository, based on your package, from GPL to BSD-3 ( https://github.com/gonum/gonum/blob/master/LICENSE) and put as part of science package on golang - https://github.com/gonum/gonum/

**Comment by DrTimothyAldenDavis** [comment]
No, CSparse/CXSparse must remain as GNU LGPL.  The same license would then apply to gonum, and I must retain copyright of the derivative work of gonum as well, for the parts of gonum that are derived from CSparse.  The details are here: https://www.copyright.gov/circs/circ14.pdf .  I can't transfer the copyright.

**Comment by Konstantin8105** [comment]
For all projects, based on your packages - GNU LGPL.
Thank you for your clarification.

Feel free for close that issue.

**Comment by DrTimothyAldenDavis** [comment]
Some of my projects are GNU GPL2, not LGPL.  It depends on the package.

I have a few BSD packages (AMD, CAMD, COLAMD, and CCOLAMD).
GraphBLAS is Apache 2.   And SuiteSparse_config has no licensing
restrictions at all.

The license depends on how the package was written, research grant support
for it, and so on.  Most are various forms of GNU licenses, but not all.

CSparse / CXSparse are GNU LGPL 2.1.

I can't make any changes to the licensing of any packages.
